src/utils/subpath-title.tsx

```diff
@@ -0,0 +1,11 @@
+export const suburlFromTitle = (title: string): string => {
```

Comment:
Alle disse funksjonene kan bruke implisitt return :)

F.eks.:
```
export const titleFromSuburl = (suburl: string): string => capitalize(suburl.split('-').join(' '));

```

Da tar alle tre funksjonene tre linjer til sammen, i stedet for å ta tre linjer hver.